lib/Backends/Interpreter/InterpreterFunction.h

```diff
@@ -74,6 +74,8 @@ class InterpreterFunction final : public CompiledFunction {
   void tearDownRuns() override;
 
   void execute() override;
+  /// Get reference to IR function.
+  IRFunction *getIR() { return F_.get(); }
```

Comment:
As above, I think adding a `collectConstants` would be better than adding `getIR` just to pass it back to runtime bundle.

Reply:
The challenge here is the CPUFunction, it doesn't keep the IR around so it would still need it passed in.

Reply:
That is a good point.  Which makes me wonder, how could we even use `collectConstants` with the CPU backend?  Since `compileIR` takes ownership of the IR and then destroys it, how can we later pass it to collect?  (Right now we only collect inside the compile function, but that's going to change soon... right?

Reply:
Yes, the device manager will have a reference to the module so that is where it can collect constants from.

Reply:
Looking at collectConstants, does it even need to take an IRFunction?  It looks like it pulls all the constants from the Module via `F->getGraph()->getParent()->getConstants()`.  It then uses `getWeightForNode` to turn those back into WeightVars but I don't see a reason for that, since all we do is call `getSizeInBytes`, which we can get without getting the WeightVar.  Am I missing something?